src/composables/use-window-scroll.ts

```diff
@@ -52,5 +58,5 @@ export function useWindowScroll({
     passive: true,
   })
 
-  return { x, y, isScrolled }
+  return Object.freeze({ x, y, isScrolled })
```

Comment:
I don't understand why this object is frozen.

Reply:
To encode the immutability of the structure? Changing the structure of this object is meaningless so why allow it? Maybe I am over thinking this though.